service/assessment/assessment.go

```diff
@@ -487,6 +487,11 @@ func (svc *Service) handleEvidence(
 			ComplianceComment:    data.Message,
 			ComplianceDetails:    data.ComparisonResult,
 			ToolId:               util.Ref(assessment.AssessmentToolId),
+			HistoryUpdatedAt:     timestamppb.Now(),
+			History: []*assessment.Record{{
+				EvidenceId:         ev.GetId(),
+				EvidenceRecordedAt: timestamppb.Now(),
+			}},
```

Comment:
That looks like creating a new list with only one item each time. I guess we want something with append, don't we?

Reply:
Yes, that’s correct. The implementation isn’t finished yet. In the next pull request I’ll add logic to decide whether the assessment result is being created for the first time or whether only its history field is being extended.

These are only preliminary preparations so far.

Reply:
#1724